types/index.d.ts

```diff
@@ -330,6 +330,45 @@ declare namespace Realm {
         readonly prototype: List<any>;
     };
 
+
+    /**
+     * Set
+     * @see { @link https://realm.io/docs/javascript/latest/api/Realm.Set.html }
+     */
+     interface Set<T> extends Collection<T> {
```

Comment:
Wouldn't it make more sense for this to be a `class` instead of an interface? (that way we can avoid lines 368 - 370)
```suggestion
     class Set<T> extends Collection<T> {
```